File: Java Basics/IntroductionToJava/src/Garden.java
import java.util.Scanner;


public class Garden {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int tomatoSeeds = input.nextInt();
		int tomatoArea = input.nextInt();
		int cucumberSeeds = input.nextInt();
		int cucumberArea = input.nextInt();
		int potatoSeeds = input.nextInt();
		int potatoArea = input.nextInt();
		int carrotSeeds = input.nextInt();
		int carrotArea = input.nextInt();
		int cabbageSeeds = input.nextInt();
		int cabbageArea = input.nextInt();
		int beansSeeds = input.nextInt();
		
		int areaLeft = 250 - (tomatoArea + cucumberArea + potatoArea + carrotArea + cabbageArea);
		double totalCost = (tomatoSeeds * 0.5) + (cucumberSeeds * 0.4) + (carrotSeeds * 0.6) + (cabbageSeeds * 0.3) +
				(potatoSeeds * 0.25) + (beansSeeds * 0.4);
		
		System.out.printf("%.2f", totalCost);
		if (areaLeft > 0) {
			System.out.println("Yes");
			System.out.println(areaLeft);
		}
		else if (areaLeft == 0) {
			System.out.println("No");
		}
		else if (areaLeft < 0) {
			System.out.println("Insufficient area");
		}
	}

}

File: C# Basics/SoftUni Exam Practice/Telerik.Garden/Garden.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Telerik.Garden
{
    class Garden
    {
        static void Main(string[] args)
        {
            double tomatoSeeds = double.Parse(Console.ReadLine());
            double tomatoArea = double.Parse(Console.ReadLine());
            double cucumberSeeds = double.Parse(Console.ReadLine());
            double cucumberArea = double.Parse(Console.ReadLine());
            double potatoSeeds = double.Parse(Console.ReadLine());
            double potatoArea = double.Parse(Console.ReadLine());
            double carrotSeeds = double.Parse(Console.ReadLine());
            double carrotArea = double.Parse(Console.ReadLine());
            double cabbageSeeds = double.Parse(Console.ReadLine());
            double cabbageArea = double.Parse(Console.ReadLine());
            double beansSeeds = double.Parse(Console.ReadLine());

            double tomatoPrice = 0.5 * tomatoSeeds;
            double cucumberPrice = 0.4 * cucumberSeeds;
            double potatoPrice = 0.25 * potatoSeeds;
            double carrotPrice = 0.6 * carrotSeeds;
            double cabbagePrice = 0.3 * cabbageSeeds;
            double beansPrice = 0.4 * beansSeeds;
            double totalPrice = tomatoPrice + cucumberPrice + potatoPrice + carrotPrice + cabbagePrice + beansPrice;

            double occupiedArea = tomatoArea + cucumberArea + potatoArea + carrotArea + cabbageArea;
            double beansArea = 250 - occupiedArea;

            Console.WriteLine("Total costs: {0:F2}", totalPrice);

            if (beansArea > 0)
            {
                Console.WriteLine("Beans area: {0}", beansArea);
                
            }
            else if (beansArea == 0)
            {
                Console.WriteLine("No area for beans");
            }
            else if (beansArea < 0)
            {
                Console.WriteLine("Insufficient area");
            }
        }
    }
}
